src/server/lock_manager/deadlock.rs

```diff
@@ -528,8 +537,8 @@ where
     /// Reconnects the leader. The leader info must exist.
     fn reconnect_leader(&mut self, handle: &Handle) {
         assert!(self.leader_client.is_none() && self.leader_info.is_some());
-        ERROR_COUNTER_VEC.reconnect_leader.inc();
-
+        LOCK_MANAGER_METRICS.with(|m| m.error_counter.reconnect_leader.inc());
```

Comment:
Looks like it would be better to separate error_counter and task_counter into different groups (because according to the code they rarely modified together).